Observer/AdyenSalesOrderClearStateDataObserver.php

```diff
@@ -0,0 +1,96 @@
+<?php
+/**
+ *                       ######
+ *                       ######
+ * ############    ####( ######  #####. ######  ############   ############
+ * #############  #####( ######  #####. ######  #############  #############
+ *        ######  #####( ######  #####. ######  #####  ######  #####  ######
+ * ###### ######  #####( ######  #####. ######  #####  #####   #####  ######
+ * ###### ######  #####( ######  #####. ######  #####          #####  ######
+ * #############  #############  #############  #############  #####  ######
+ *  ############   ############  #############   ############  #####  ######
+ *                                      ######
+ *                               #############
+ *                               ############
+ *
+ * Adyen Payment Module
+ *
+ * Copyright (c) 2021 Adyen N.V.
+ * This file is open source and available under the MIT license.
+ * See the LICENSE file for more info.
+ *
+ * Author: Adyen <magento@adyen.com>
+ */
+
+namespace Adyen\Payment\Observer;
+
+use Adyen\Payment\Helper\StateData;
+use Adyen\Payment\Logger\AdyenLogger;
+use Magento\Framework\Event\Observer;
+use Magento\Framework\Event\ObserverInterface;
+use Magento\Quote\Api\CartRepositoryInterface;
+use Magento\Quote\Model\ResourceModel\Quote\Payment as QuotePaymentResourceModel;
+use Magento\Sales\Model\ResourceModel\Order\Payment as OrderPaymentResourceModel;
+
+class AdyenClearStateDataObserver implements ObserverInterface
```

Comment:
The filename does not match with the classname causing an autoload error that breaks the checkout. It needs to be renamed:

Observer/AdyenSalesOrderClearStateDataObserver.php -> Observer/AdyenClearStateDataObserver.php